requirements.in

```diff
@@ -16,3 +16,4 @@ psycopg2-binary
 datapunt-objectstore
 drf-amsterdam
 xmltodict
+setuptools  # Needed to solve Trivy security risk
```

Comment:
Is deze nodig in prod? Volgens mij voldoende als hij in requirements_dev.in zit toch?

Reply:
Nee hij is nodig op productie om een security risk van Trivy te resolven